1-Draft/RFCXXXX-Obsoletion-and-Deprecation.md

```diff
@@ -220,20 +230,24 @@ This has four possible values:
   - Warning messages will not be shown
 - Continue
   - Warning messages will be shown
-- Stop
-  - Warning messages will be converted to terminating error messages
 - NotAvailable
   - Any obsolete item will be not found.
-    Cmdlets, parameters, Types, and members will produce an appropriate "NotFound" error.
+   This means that cmdlets will produce a CommandNotFound error and types (or their obsolete members) will produce a runtime error.
+   Additionally, the warning message will not be produced.
+- Stop
+  - Warning messages will be converted to terminating error messages
 
+---
+**NOTE**
 A ubiquitous parameter could be added, but this seems less interesting as the user must have _a priori_ knowledge about the obsolete cmdlet, thus diluting its usefulness.
 It may be possible to use a ubiquitous parameter via `PSDefaultParameterValues`, but this feels like _additional_ complexity rather than simplification.
```

Comment:
Similar comments to prior comment regarding use of "ubiquitous". There is no usage of "ubiquitous" in current help documents. Variables of this type are referred to as "preference variables".